tests/test_acl_portchannel.py

```diff
@@ -0,0 +1,190 @@
+from swsscommon import swsscommon
+
+import time
+
+class TestPortChannelAcl(object):
+    def setup_db(self, dvs):
+        self.pdb = swsscommon.DBConnector(0, dvs.redis_sock, 0)
+        self.adb = swsscommon.DBConnector(1, dvs.redis_sock, 0)
+        self.cdb = swsscommon.DBConnector(4, dvs.redis_sock, 0)
+
+    def create_port_channel(self, dvs, alias):
+        tbl = swsscommon.Table(self.cdb, "PORTCHANNEL")
+        fvs = swsscommon.FieldValuePairs([("admin_status", "up"),
+                                          ("mtu", "9100")])
+        tbl.set(alias, fvs)
+        time.sleep(1)
+
+    def remove_port_channel(self, dvs, alias):
+        tbl = swsscommon.Table(self.cdb, "PORTCHANNEL")
+        tbl._del(alias)
+        time.sleep(1)
+
+    def add_port_channel_members(self, dvs, lag, members):
+        tbl = swsscommon.Table(self.cdb, "PORTCHANNEL_MEMBER")
+        fvs = swsscommon.FieldValuePairs([("NULL", "NULL")])
+        for member in members:
+            tbl.set(lag + "|" + member, fvs)
+            time.sleep(1)
+
+    def remove_port_channel_members(self, dvs, lag, members):
+        tbl = swsscommon.Table(self.cdb, "PORTCHANNEL_MEMBER")
+        for member in members:
+            tbl._del(lag + "|" + member)
+            time.sleep(1)
+
+    def create_acl_table(self, dvs, table_name, ports):
+        tbl = swsscommon.Table(self.cdb, "ACL_TABLE")
+        fvs = swsscommon.FieldValuePairs([("POLICY_DESC", "LAG_ACL_TEST"),
+                                          ("TYPE", "L3"),
+                                          ("PORTS", ports)])
+        tbl.set(table_name, fvs)
+        time.sleep(1)
+
+    def remove_acl_table(self, dvs, table_name):
+        tbl = swsscommon.Table(self.cdb, "ACL_TABLE")
+        tbl._del(table_name)
+        time.sleep(1)
+
+    def create_acl_rule(self, dvs, table_name, rule_name):
+        tbl = swsscommon.Table(self.cdb, "ACL_RULE")
+        fvs = swsscommon.FieldValuePairs([("PRIORITY", "88"),
+                                          ("PACKET_ACTION", "FORWARD"),
+                                          ("L4_SRC_PORT", "8888")])
+        tbl.set(table_name + "|" + rule_name, fvs)
+        time.sleep(1)
+
+    def remove_acl_rule(self, dvs, table_name, rule_name):
+        tbl = swsscommon.Table(self.cdb, "ACL_RULE")
+        tbl._del(table_name + "|" + rule_name, fvs)
+        time.sleep(1)
+
+    # Frist create port channel
+    # Second create ACL table
+    def test_PortChannelAcl(self, dvs):
```

Comment:
>test_PortChannelAcl [](start = 8, length = 19)

test_PortChannelBeforeAcl

and 

test_PortChannelAfterAcl

try to give meaningful name to function